packages/tobot_math/tests/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# include google test
+include(FetchContent)
+FetchContent_Declare(
+  googletest
+  URL https://github.com/google/googletest/archive/03597a01ee50ed33e9dfd640b249b4be3799d395.zip
+)
+
+# For Windows: Prevent overriding the parent project's compiler/linker settings
+set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
+FetchContent_MakeAvailable(googletest)
+
+set(TOBOT_MATH_TESTS tobot-tooling_TESTS)
```

Comment:
Why tooling test 🗡️

Reply:
Additionally adapt the name in the test.yaml file

Reply:
Fixed with https://github.com/jksevend/tobot-engine/pull/47/commits/0ec2e6c749992ec5fbaa43cbc8e8822871ed09ff